### ISSUE TITLE
PouchDB error: you must install a SQLite plugin ... with cordova windows

### ISSUE THREAD

**Issue by dozer247** [pouchdb-community/pouchdb-adapter-cordova-sqlite#75]
I'm getting the following error when running my app using the cordova windows platform.

```
PouchDB error: you must install a SQLite plugin in order for PouchDB to work on this platform. Options:
- https://github.com/nolanlawson/cordova-plugin-sqlite-2'
- https://github.com/litehelpers/Cordova-sqlite-storage'
- https://github.com/Microsoft/cordova-plugin-websql'
```
For the windows platform i'm using the cordova-sqlite-storage 2.0.4 plugin.

I'm getting the error message because the openDatabase method is missing as i believe window edge doesn't support webSQL

But in saying that it does look like it is using the sqliteplugin as i seen the following in the log followed by a SQLite3 step error.

```
OPEN database: <emailaddress>
new transaction is waiting for open operation
open db name: <emailaddress> at full path: C:\Users\Darrell\AppData\Local\Packages\com.oneplaceonline.FranchiseMgr_h35559jr9hy9m\LocalState\<emailaddress>
new transaction is waiting for open operation
OPEN database: <emailaddress> - OK
DB opened: <emailaddress>
sql exception error: SQLite3 step error result code: 1


```

Does PouchDB with  an sqlite plugin work on the cordova windows platform?

i'm using
cordova 7.0.1
cordova windows@5.0.0
PouchDB 6.0.7
I'm running the app on a  windows 10 PC as a uwp app.

**Comment by fxnn** [comment]
The error message doesn't stop the plugin to work. And actually, I think that the conditions under which the message appears should be made more strict.

Currently, the error message is shown when the expression

    typeof cordova === 'undefined' || typeof sqlitePlugin === 'undefined' || typeof openDatabase === 'undefined'

evaluates to `true`. But in fact, the plugin works as soon as `sqlitePlugin` _or_ `openDatabase` exists, and it seems to me that it doesn't even use the `cordova` object.

I wonder whether it would be better when the error message would only be shown if

    typeof sqlitePlugin === 'undefined' && typeof openDatabase === 'undefined'

is `true` -- note the `&&` instead of `||`.

**Comment by CarlosAyala** [comment]
Hi,
The cordova-sqlite-storage have a lot of issues with Windows, check the readme in "Windows platform notes" in the github repo for this cordova plugin.

**Comment by netstart** [comment]
I have the same problem on a Linux Mint

**Comment by paolosanchi** [comment]
fixed in v2.0.7
